client/client.go

```diff
@@ -827,6 +811,7 @@ func (ec *SDKClient) GetGasPrice(
 			newGasPrice.Int(gasPrice)
 		}
 	} else {
+		log.Println("Setting existing gas price")
```

Comment:
we should probably eventually introduce structured logging via [zap](https://github.com/uber-go/zap) - add it as an issue.

Reply:
https://jira.coinbase-corp.com/browse/ROSE-567